.github/workflows/vib-publish.yaml

```diff
@@ -0,0 +1,113 @@
+name: 'VIB Publish'
```

Comment:
In order to follow [the convention](https://github.com/bitnami/charts/blob/master/.github/workflows/vib-verify.yaml#L1) `name: 'VIB'`

Reply:
I noticed that change and hence renamed it to this from `vib-publish`, but then noticed that would create 2 duplicated entries at the [Actions](https://github.com/bitnami/charts/actions) page (note there are already 3: 1 for vib-verify.yaml workflow + the 2 extra workflows).
Do you still think we should name this `VIB` only? Maybe we could name them `VIB - Publish` and `VIB - PR Verify`?

Reply:
@aruiz14 I like your proposal:

- `VIB - PR verify` for verifying the contribution
- `VIB - Publish` for producing final bits and publishing them